_posts/2025-08-26-strimzi-metrics-reporter.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Using Strimzi Metrics Reporter to Expose Kafka Metrics in Prometheus Format"
+date: 2025-10-01
+author: Owen Corrigan
+---
+When deploying new Kafka cluster it may be one of your intentions to monitor the cluster using metrics.
+`Apache Kafka®` generates and exposes metrics that reflect how a system is operating. 
```

Comment:
What about "how it is operating" instead of "how a system is operating", I mean it's exposing metrics about itself, not about other systems.

Reply:
It's probably best to not format "Apache Kafka®" as code. Also use the full name on the first instance on the line above instead of here.

Reply:
Refactored